pkg/chartutil/create.go

```diff
@@ -189,6 +190,10 @@ spec:
         app.kubernetes.io/name: {{ include "<CHARTNAME>.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      {{- with .Values.imagePullSecrets }}
```

Comment:
Match the existing indentation style.

```suggestion
    {{- with .Values.imagePullSecrets }}
```